parachain/Cargo.toml

```diff
@@ -12,6 +12,6 @@ members = [
     "pallets/ethereum-beacon-client",
     "pallets/control",
     "pallets/control/runtime-api",
+    "runtime/tests",
```

Comment:
The tests crate is only used by the runtimes in `polkadot-sdk/`, so I don't think we should add it to the workspace.

Also because its going to bloat our local compile times.